src/txgraph.h

```diff
@@ -94,9 +94,10 @@ class TxGraph
     virtual void SetTransactionFee(const Ref& arg, int64_t fee) noexcept = 0;
 
     /** TxGraph is internally lazy, and will not compute many things until they are needed.
-     *  Calling DoWork will compute everything now, so that future operations are fast. This can be
-     *  invoked while oversized. */
-    virtual void DoWork() noexcept = 0;
+     *  Calling DoWork will perform some work now (controlled by iters) so that future operations
+     *  are fast, if there is any. Returns whether all currently-available work is done. This can
+     *  be invoked while oversized, but oversized graphs will be skipped by this call. */
+    virtual bool DoWork(uint64_t iters) noexcept = 0;
```

Comment:
nit: `maximum_iterations` would be clearer than `iters`?

This is nice  the mempool can decide what to pass to `DoWork` by converting the time it can spend into iterations.

Reply:
`max_iters` suffices, if you end up touching the PR